resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStep.scala

```diff
@@ -216,6 +216,11 @@ private[spark] class BasicExecutorFeatureStep(
         .endSpec()
       .build()
 
+    val schedulerName = kubernetesConf.get(KUBERNETES_EXECUTOR_SCHEDULER_NAME)
+    if (schedulerName.nonEmpty) {
```

Comment:
This is fine. Also: `kubernetesConf.get(KUBERNETES_EXECUTOR_SCHEDULER_NAME).foreach(executorPod.getSpec.setSchedulerName)`

Reply:
Resolved, thx Sean~